tgui/packages/tgui/interfaces/Safe.js

```diff
@@ -0,0 +1,149 @@
+import { Fragment } from 'inferno';
+import { resolveAsset } from '../assets';
+import { useBackend } from '../backend';
+import { Box, Button, Icon, Section } from '../components';
+import { Window } from "../layouts";
+
+export const Safe = (properties, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    dial,
+    open,
+  } = data;
+  return (
+    <Window
+      width={625}
+      height={760}
+      theme="ntos">
+      <Window.Content>
+        <Box className="Safe__engraving">
+          <Dialer />
+          <Box>
+            <Box
+              className="Safe__engraving-hinge"
+              top="25%" />
+            <Box
+              className="Safe__engraving-hinge"
+              top="75%" />
+          </Box>
+          <Icon
+            className="Safe__engraving-arrow"
+            name="long-arrow-alt-down"
+            size="3"
+          /><br />
+          {open ? (
+            <Contents />
+          ) : (
+            <Box
+              as="img"
+              className="Safe__dial"
+              src={resolveAsset('safe_dial.png')}
+              style={{
+                "transform": "rotate(-" + (3.6 * dial) + "deg)",
+              }}
+            />
+          )}
+        </Box>
+        {!open && (
+          <Help />
+        )}
+      </Window.Content>
+    </Window>
+  );
+};
+
+const Dialer = (properties, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    dial,
+    open,
+    locked,
+    broken,
+  } = data;
+  const dialButton = (amount, right) => {
+    return (
+      <Button
+        disabled={open || (right && !locked) || broken}
+        icon={"arrow-" + (right ? "right" : "left")}
+        content={(right ? "Right" : "Left") + " " + amount}
+        iconPosition={right}
```

Comment:
You forgot to specify "left", "right" here.

Reply:
Fixed.